sirepo/auth_db/email.py

```diff
@@ -39,6 +39,12 @@ class AuthEmailUser(sirepo.auth_db.UserDbBase):
     token = sqlalchemy.Column(sqlalchemy.String(sirepo.util.TOKEN_SIZE), unique=True)
     expires = sqlalchemy.Column(sqlalchemy.DateTime())
 
+    def __init__(self, *args, **kwargs):
+        for x in ("unverified_email", "user_name"):
+            if kwargs.get(x):
```

Comment:
I think an explicit null check is more robust here:

```py
if (v := kwargs.get(x)) is not None:
    kwargs[x] = v.lower()
```

Reply:
[3a9f0d1](https://github.com/radiasoft/sirepo/pull/7027/commits/3a9f0d1399801354cea9e77a661893218636e2dd)